index.html

```diff
@@ -1,8 +1,6 @@
 <!DOCTYPE html>
-<!--[if lt IE 8]><html class="lt-ie10 lt-ie9 lt-ie8" lang="nl"><![endif]-->
-<!--[if IE 8]><html class="lt-ie10 lt-ie9" lang="nl"><![endif]-->
-<!--[if IE 9]><html class="lt-ie10" lang="nl"><![endif]-->
-<!--[if gt IE 9]><!--><html lang="nl"><!--<![endif]-->
+<!--[if lt IE 10]><html class="lt-ie10" lang="nl"><![endif]-->

```

Comment:
Goed dat je deze hebt aangepast, maar daarbij vraag ik me meteen af of we de `lt-ie10` class daadwerkelijk ook nog nodig gaan hebben? Die target alleen IE9 en lager en die ondersteunen we officieel ook niet meer.